MediaManager/Plugin.MediaManager.iOS/AudioPlayerImplementation.cs

```diff
@@ -218,7 +218,10 @@ private void InitializePlayer()
             if(mediaFile != null)
                 nsUrl = new NSUrl(mediaFile.Url);
 
-            if (Status == MediaPlayerStatus.Paused)
+            var currentPlayerAsset = (AVUrlAsset) Player.CurrentItem?.Asset;
+            var currentPlayerItemUrl = currentPlayerAsset?.Url?.ToString();
+
+            if (Status == MediaPlayerStatus.Paused && mediaFile.Url == currentPlayerItemUrl)
```

Comment:
The only problem I could imagine is if you have different mediaFiles with the same Url, which in my opinion should never happen. What else would you do here? Save the instance in a private field and compare it later? Or is there something else we can use for identification?